qa327/backend.py

```diff
@@ -42,12 +43,70 @@ def register_user(email, name, password, password2):
 
     hashed_pw = generate_password_hash(password, method='sha256')
     # store the encrypted password rather than the plain password
-    new_user = User(email=email, name=name, password=hashed_pw)
+    new_user = User(email=email, name=name, password=hashed_pw, balance=5000)
 
     db.session.add(new_user)
     db.session.commit()
     return None
 
+def get_ticket(ticket_id):
+    """
+    Get a ticket by a given id
+    :param ticket_id: the id of the ticket
+    :return: a ticket that has the matched id
+    """
+    ticket = Ticket.query.filter_by(id=ticket_id).first()
+    return ticket
+
+def update_ticket(ticket_id, name, quantity, price, date):
+    """
+    Update ticket info in the database
+    :param ticket_id: the id of the ticket to be updated
+    :param name: the name of the ticket
+    :param quantity: the amount of tickets for sale
+    :param price: the price of the ticket
+    :param date: the expiry date of the ticket
+    :return: an error message if there is any, or None if register succeeds
+    """
+    ticket = get_ticket(ticket_id)
+    ticket.name = name
+    ticket.quantity = quantity
+    ticket.price = price
+    ticket.date = date
+    db.session.commit()
+
+def buy_ticket(ticket_id, buyer_id):
```

Comment:
I would add docstrings for this function too, to match the `update_ticket` and `sell_ticket` functions that also have docstrings